tutorials/optical_tactile_plugin.md

```diff
@@ -0,0 +1,171 @@
+
+Currently, the plugin is required to work within models that have one link, one
+contact sensor and one depth camera. In order to simulate the behaviour of a
+deformable membrane that returns the contact forces of the object being touched, 
+we merge the information coming from this two different sensors. Vectors normal to 
+the surfaces being touched are computed from the values coming from the depth camera.
+Next, this values are merged with the information returned by the contact sensor, i.e.
+force magnitudes, penetration and depth.
+
+Functionality can be added on top of the information returned by this sensor,
+like object recognition, tactile imaging, precise manipulation or human-robot
+interaction. However, this is something not currently supported, and the plugin
+only returns the information computed.
+
+The plugin allows the user to visualize the surface vectors of the objects being touched.
+In the next image, we can see these vectors when we touch the ring and the top
+surface of a can of coke:
+
+<img src="https://github.com/ignitionrobotics/ign-gazebo/blob/master/tutorials/files/optical_tactile_plugin/coke_can.png"/>
+
+## Components layout
+
+As described in the previous section, there are several components that make up 
+the optical tactile sensor. The following graphical description aims to help the
+user get a better understanding of the different components of the plugin:
+
+```
+                               ___
+                             /|   |
+                            / |   |
+                           /  |   |
+                        _ /   |   |
+ Depth camera ---------|_|    | ·-|----- Model's origin
+ placed behind the        \   |   |
+ 'membrane' (see the next  \  |   |
+ section)                   \ |   |
+                             \|___|--- 'Tactile membrane': This is the collision
+                                        geometry of the contact sensor. Forces
+                                        returned by the sensor are inside this
+                                        volume (check the <extended_sensing>
+                                        parameter)
+```
+
+## Assumptions
+
+The following assumptions have been made when working on this plugin:
+
+- Currently, the plugin's model should be made up of one link, one contact
+sensor and one depth camera.
+- The geometry of the sensor must be a box. Ideally, the size should be around
+20x20x5 mm, but this can be changed.
+- The depth camera should be displaced from the model's origin in the negative
+direction of the X axis, so it's not placed inside the membrane. This can be done
+setting the depth camera's `<pose>` accordingly. Check out the [example below](#try-it-out).
+- There should not be a `<visual>` element inside the link, so the depth camera
+isn't facing the link and can return coherent values. The 'membrane' can still
+be visualized setting the `<visualize_sensor>` parameter to true. For more information
```

Comment:
I think this is more clear: "so the depth camera... values" -> "so that the link of the sensor does not block the depth camera's view"

Word missing: visualized `by` setting